### PR TITLE
Add Followers component tests

### PR DESCRIPTION
**Pull request by ehnat0n**
PR covers the story for Followers component: https://amonteschool.atlassian.net/browse/GTHB004-16

Known bugs:
1. Misleading information in "Followers" on page load/refresh. Bug report: https://amonteschool.atlassian.net/browse/GTHB004-92
2. Followers component capitalizes first letter of every follower listed. Bug report: https://amonteschool.atlassian.net/browse/GTHB004-93
3. Delayed user info update after refresh (same bug affects User Info component). https://amonteschool.atlassian.net/browse/GTHB004-91

### CODE REVIEW

File: pom/components/base_element.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def find_element(self, locator):
     def find_visible_element(self, locator):
         """Returns WebElement with specified locator after explicit wait
         for element being visible."""
-        return self.wait.until(ec.presence_of_element_located(locator))
+        return self.wait.until(ec.visibility_of_element_located(locator))
```
Comment (galiur) [inline pom/components/base_element.py:25]: Why do you use 2 'return' statements here? Can't you just use:
`return self.wait.until(ec.visibility_of_element_located(locator))`
cause this method already checks for presence of a element on the page

File: pom/components/base_element.py [hunk 2]
```diff
@@ -32,7 +32,7 @@ def find_elements(self, locator):
     def find_visible_elements(self, locator):
         """Returns all WebElements with specified locator after explicit wait
         for elements being visible."""
-        return self.wait.until(ec.presence_of_all_elements_located(locator))
+        return self.wait.until(ec.visibility_of_all_elements_located(locator))
```
Comment (galiur) [inline pom/components/base_element.py:35]: Same questions here

**Review by galiur — CHANGES_REQUESTED** [review]
Ok, I couldn't understand everything that you did but I tried to do my best! ) Cause I'm still not that good at so many topics. It's hard to read even your own code sometimes.

File: pom/components/base_element.py (hunk 1 above)
Comment (ehnat0n) [reply pom/components/base_element.py:25]: Yeah. I created 2nd method to have both options - wait for element presence and for element visibility (on top of presence). But just copy pasted the code originally and forgot to change the wait.

So in this commit I replaced presence with visibility for second set of methods. Now there is one "find_element" that just uses the wait for presence. And this new "find_visible_element" that waits for visibility.

File: pom/components/base_element.py (hunk 2 above)
Comment (ehnat0n) [reply pom/components/base_element.py:35]: Same as above - red one removed because was copy paste and green one added instead.

**Review by galiur — APPROVED** [review]

File: features/environment.py [hunk 3]
```diff
@@ -38,5 +39,6 @@ def before_feature(context, feature):
     # Creating instances to work with app components
```
Comment (In-Anna) [inline features/environment.py:39]: Haven't found a usage of context.wait anywhere in steps (and was not expecting).
Better to move this WebDriverWait(context.browser, EXPLICIT_WAIT) to Base and you might not need context.wait at all

File: features/environment.py [hunk 4]
```diff
@@ -38,5 +39,6 @@ def before_feature(context, feature):
...
     context.search_field_element = SearchField(context)
     context.user_info_element = UserInfo(context)
+    context.user_followers_element = Followers(context)
```
Comment (In-Anna) [inline features/environment.py:42]: No need to create instances here - only where you need it

**Review by In-Anna — APPROVED** [review]

**Pull request merged**